### PR TITLE
Break the kubetest2 when the terraform apply fails

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
Adding lines of code to exit the flow when the deployer fails terraform.apply.
Below is the task:
https://github.ibm.com/powercloud/container-dev/issues/1324

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/24#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: kubetest2-tf/deployer/deployer.go
Comment (mkumatag) [inline kubetest2-tf/deployer/deployer.go:134]: oops, lets have a flag to make this decision :) lets not exit by default.

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: kubetest2-tf/deployer/deployer.go
Comment (Rajalakshmi-Girish) [reply kubetest2-tf/deployer/deployer.go:141]: Done.

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/24#issuecomment-736997161" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/24#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
